packages/angular_devkit/core/src/json/schema/interface.ts

```diff
@@ -109,7 +109,7 @@ export interface PromptDefinition {
 }
 
 export type PromptProvider = (definitions: Array<PromptDefinition>)
-  => SubscribableOrPromise<{ [id: string]: JsonValue }>;
+  => SubscribableOrPromise<{ [id: string]: JsonValue } | { [id: string]: JsonValue }[]>;
```

Comment:
This shouldn't need to be changed.  Each `id` (i.e., field) should have one value (a `JsonValue` can be an array).

Reply:
That makes sense. This was a bit of an artifact from some typing errors I had caused while debugging. Good catch!